db/seeds.rb

```diff
@@ -22,17 +22,13 @@
 user9 = User.create!(id: 19, first_name: 'Ethan', last_name: 'Doe', major: 'MATH', email: "ethan@tamu.edu", password: "T3st!!i")
 
 
-#role_id 1 is tutor, 2 is student, left 0 open for admin during testing purposes
-role5 = Role.create!(id: 15, role_id: 1)
-role6 = Role.create!(id: 16, role_id: 1)
-role7 = Role.create!(id: 17, role_id: 2)
-role8 = Role.create!(id: 18, role_id: 1)
-role9 = Role.create!(id: 19, role_id: 2)
-
 TutoringSession.create(id: 1, tutor_id: user2.id, scheduled_datetime: Time.now, completed_datetime: 0, session_status: "")
 TutoringSession.create(id: 2, tutor_id: user3.id, scheduled_datetime: Time.now, completed_datetime: 0, session_status: "")
-=end
-Role.create(role_name: 'Admin');
-Role.create(role_name: 'Tutor');
+
+admin_role = Role.create(role_name: 'Admin');
+tutor_role = Role.create(role_name: 'Tutor');
 Role.create(role_name: 'Student');
 Role.create(role_name: 'Spartan Tutor');
+
+user5.roles << tutor_role
+user8.roles << tutor_role
```

Comment:
Pretty much none of this file should be here, aside from the role creation. I'm not sure why all the old stuff was uncommented, but it needs to be either commented back out or deleted.

Reply:
I commented it out